src/encryption/wallet.ts

```diff
@@ -149,17 +155,14 @@ function decryptLegacy(dataBuffer: Buffer,
  */
 export async function decryptMnemonic(data: (string | Buffer), 
                                       password: string, 
-                                      triplesecDecrypt: TriplesecDecryptSignature
+                                      triplesecDecrypt?: TriplesecDecryptSignature
 ): Promise<string> {
   const dataBuffer = Buffer.isBuffer(data) ? data : Buffer.from(data, 'hex')
-  try {
-    return await decryptMnemonicBuffer(dataBuffer, password)
-  } catch (err) {
-    // If it was a password error, don't even bother with legacy
-    if (err instanceof PasswordError) {
-      throw err
-    }
+
+  if (triplesecDecrypt) {
```

Comment:
The `triplesecDecrypt` arg exists so that caller apps can be responsible for including the triplesec lib, and can then pass the triplesec decrypt function to blockstack.js. 

This usage looks like "if triplesecDecrypt exists then do legacy" -- which won't work